internal/domain/pycompat_fuzz_test.go

```
@@ -0,0 +1,59 @@
+	f.Add(strings.Repeat("x", 1<<16)) // 64KiB
+	f.Fuzz(func(t *testing.T, s string) {
+		h1 := HashText(s)
+		h2 := HashText(s)
+		if h1 != h2 {
+			t.Fatalf("同输入两次 hash 不等: %s / %s", h1, h2)
+		}
+		if len(h1) != 16 {
+			t.Fatalf("hash 长度 %d ≠ 16: %q", len(h1), h1)
+		}
+		for _, c := range h1 {
+			if !(c >= '0' && c <= '9' || c >= 'a' && c <= 'f') {
```

golangci-lint (2.12.2)

[error] QF1001: could apply De Morgan's law

(staticcheck)